pkg/controller/v1alpha2/applicationconfiguration/render.go

```diff
@@ -251,7 +257,8 @@ func setTraitProperties(t *unstructured.Unstructured, traitName, namespace strin
 }
 
 // SetWorkloadInstanceName will set metadata.name for workload CR according to createRevision flag in traitDefinition
-func SetWorkloadInstanceName(traitDefs []v1alpha2.TraitDefinition, w *unstructured.Unstructured, c *v1alpha2.Component) error {
+func SetWorkloadInstanceName(traitDefs []v1alpha2.TraitDefinition, w *unstructured.Unstructured, c *v1alpha2.Component,
+	currentWorkload *unstructured.Unstructured) error {
```

Comment:
```suggestion
	existingWorkload *unstructured.Unstructured) error {
```

I prefer the name `existingWorkload`, because it's hard to define who is current？